CovidWatch/app/src/main/res/layout/fragment_main.xml

```diff
@@ -203,7 +203,7 @@
             android:background="@color/primaryBackground"
             android:paddingStart="10dp"
             android:paddingEnd="10dp"
-            android:text="Self Report"
+            android:text="@string/self_report_text"
```

Comment:
this is the same value as line 185, but probably meant to be the one from line 159?
```suggestion
            android:text="@string/self_report"
```